examples/nidaqmx_analog_input/_nidaqmx_helpers.py

```diff
@@ -11,7 +11,7 @@
 def create_task(
     session_info: nims.session_management.SessionInformation,
     session_grpc_channel: Optional[grpc.Channel] = None,
-    initialization_behavior=nidaqmx.SessionInitializationBehavior.AUTO,
+    initialization_behavior: nidaqmx.SessionInitializationBehavior = nidaqmx.SessionInitializationBehavior.AUTO,
```

Comment:
I was surprised to learn that mypy doesn't infer the parameter type from its default value.